packages/components/inputs/localized-rich-text-input/src/editor.tsx

```diff
@@ -239,13 +319,13 @@ const renderEditor = (props, editor, next) => {
     ...filterDataAttributes(props),
   };
 
-  const isFocused = props.editor.value.selection.isFocused;
+  const isFocused = props.editor?.value.selection.isFocused;
 
   return (
     <Editor editor={editor} {...passedProps}>
       {children}
       <HiddenInput
-        isFocused={isFocused}
+        isFocused={Boolean(isFocused)}
```

Comment:
`isFocused` seems like it's already a boolean from `editor` props but it's been casted here.

Reply:
`isFocused` is required on HiddenInput on optional on Editor. That is why casting is necessary there.

Reply:
Oh, thanks!